src/Audit/Drupal/SolrVersion.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Drutiny\Audit\Drupal;
+
+use Drutiny\Audit;
+use Drutiny\Sandbox\Sandbox;
+use Drutiny\Annotation\Param;
+
+/**
+ * Check the version of Drupal project in a site.
+ * @Param(
+ *  name = "module",
+ *  description = "The module to version information for",
+ *  type = "string"
+ * )
+ * @Param(
+ *  name = "version",
+ *  description = "The static version to check against.",
+ *  type = "string"
+ * )
+ * )
+ */
+class SolrVersion extends Audit
+{
+    /**
+     * @inheritdoc
+     */
+    public function configure()
+    {
+        $this->addParameter(
+            'module',
+            static::PARAMETER_OPTIONAL,
+            'Module to check.'
+        );
+        $this->addParameter(
+            'version',
+            static::PARAMETER_OPTIONAL,
+            'Module to check.'
+        );
+    }
+
+    public function audit(Sandbox $sandbox)
+    {
+        $module = $sandbox->getParameter('module');
```

Comment:
$sandbox is deprecated. Should use $this->getParameter instead.

Reply:
Actually, should module even be a parameter here? seems like the logic is pretty specific to a solr module?